pyecharts/custom/page.py

```diff
@@ -34,10 +34,14 @@ def render(self,
                template_name='simple_page.html',
                object_name='page',
                extra_context=None):
-        context = {object_name: self}
-        context.update(extra_context or {})
-        html = engine.render(template_name, **context)
-        utils.write_utf8_html_file(path, html)
+        env = engine.create_default_environment()
+        env.render_chart_to_file(
```

Comment:
this method needs improvement. page and other charts inherit from chart, but relying object_name to switch looks like a comprise.